GeneratorInterface/RivetInterface/python/genParticles2HepMC_cfi.py

```diff
@@ -2,5 +2,5 @@
 
 generator = cms.EDProducer("GenParticles2HepMCConverter",
     genParticles = cms.InputTag("genParticles"),
-    genEventInfo = cms.InputTag("generator"),
+    genEventInfo = cms.InputTag("generator", "", "SIM"),
```

Comment:
Without specifying the process name, it raises some warning because this module tries to read object with same name:
%MSG-e UnrunnableSchedule:  AfterModConstruction 31-Jul-2015 14:51:13 CEST  pre-events
Module run order problem found: 
```generator consumes generator
 Running in the threaded framework would lead to indeterminate results.
 Please change order of modules in mentioned Path(s) to avoid inconsistent module ordering.
%MSG```
Of course one can change the name of GenParticles2HepMCConverter module to avoid such overlap, but in this case the input label of RivetAnalyzer have to be changed. It's just a different way to workaround the problem.